core/helpers.py

```diff
@@ -11,6 +11,13 @@
 )
 
 
+def instantiate_subclass(base_class, subclass_name):
```

Comment:
This function is not instantiating anything. I think something like `def get_subclass` would be more accurate. Also, please add a small dosctring explaining what this does.

Reply:
fixed in the new commit